docs/timelion/getting-started/timelion-math.asciidoc

```diff
@@ -0,0 +1,61 @@
+[[timelion-math]]
+=== Using mathematical functions
+
+You’ve learned how to create and style a Timelion visualization in the previous two sections. This section will explore the mathematical functions Timelion offers. You will continue to use the https://www.elastic.co/downloads/beats/metricbeat[Metricbeat data] to create a new Timelion visualization for inbound and outbound network traffic. To start, you'll need to add a new Timelion visualization to the sheet.
+
+In the top menu, click `Add` to add a second visualization. When added to the sheet, you’ll notice that the query bar has been replaced with the default `.es(*)` expression. This is because the query is associated with the visualization on the Timelion sheet you have selected. 
+
+image::images/timelion-math01.png[]
+{nbsp}
+
+To start tracking the inbound / outbound network traffic, your first expression will calculate the maximum value of `system.network.in.bytes`. Enter the expression below into your Timelion query bar:
+
+[source,text]
+----------------------------------
+.es(index=metricbeat*, timefield=@timestamp, metric=max:system.network.in.bytes)
+----------------------------------
+
+image::images/timelion-math02.png[]
+{nbsp}
+
+Now that you have the maximum value for the inbound traffic, let’s calculate the derivative to chart the change over time. The `derivative()` function is used to plot the change in values over time. This can be easily done by appending the `.derivative()` to the end of an expression. Use the following expression to update your visualization:
```

Comment:
I agree, though I've been told that we shouldn't using "we" and "we will" in the tutorial. How's this?

>Monitoring network traffic is much more valuable when plotting the rate of change. The `derivative()` function is used do just that - plot the change in values over time. This can be easily done by appending the `.derivative()` to the end of an expression.